src/sheet-native.tsx

```diff
@@ -88,25 +88,36 @@ const showActionsListNativeSheet = (
 
 const showInfoNativeSheet = async (
     nativeSheetImplementation: NativeSheetImplementation,
-    {title, subtitle, description, items}: SheetPropsByType['INFO']
+    {title, subtitle, description, items, button}: SheetPropsByType['INFO']
 ): Promise<SheetResultByType['INFO']> => {
-    // nothing to return, this is an informative sheet
-    await (nativeSheetImplementation as NativeSheetImplementation)({
+    const infoSheetContent = {
+        type: 'LIST' as const,
+        id: 'list-0',
+        listType: 'INFORMATIVE' as const,
+        autoSubmit: false,
+        selectedIds: [],
+        items,
+    };
+
+    return await (nativeSheetImplementation as NativeSheetImplementation)({
         title,
         subtitle,
         // TODO: add multiline support to native sheet
         description: normalizeDescriptionForNative(description),
-        content: [
-            {
-                type: 'LIST',
-                id: 'list-0',
-                listType: 'INFORMATIVE',
-                autoSubmit: false,
-                selectedIds: [],
-                items,
-            },
-        ],
-    });
+        content: button
```

Comment:
in the tests you and Pedro did in Android webviews, this makes old apps return an error (fix pending by Android team). Could we capture that error here and fallback to showing the sheet without button?

Reply:
Discussed offline. We can't do such thing, because the Android error occurs when pressing the button, not when opening the sheet.

This should be handled by the app itself